### PR TITLE
CA-26/Basic event details screen UI impl

### PR DESCRIPTION
**Pull request by rock3r**
This PR adds the first bits to have an event details:

 * Click listener for schedule items
 * EventDetailsActivity's base layout (no logic nor real data views yet) with LOADS of TODOs on extracting values and all, this is just so we can parallelise data fetching and UI impl from this point onwards

scrolly

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -71,7 +71,7 @@ android {
 
         resValueString 'api_value_base_url', (buildProperties.secrets['baseUrl'] | buildProperties.env['BASE_URL']).string
 
-        resValueString 'social_query', (buildProperties.application['socialQuery'] | '#AndroidDev').string
+        resValueString 'social_query', (buildProperties.application['socialQuery'] | "#AndroidDev").string
```
Comment (tiwiz) [inline app/build.gradle:74]: Shouldn't it be "#DroidconIT"?

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -81,6 +81,9 @@
       android:name="io.fabric.ApiKey"
       android:value="${fabricApiKey}" />
 
+    <activity
+      android:name="com.connfa.eventdetails.EventDetailsActivity"
+      android:theme="@style/Theme.Connfa.EventDetails" />
```
Comment (tiwiz) [inline app/src/main/AndroidManifest.xml:86]: Maybe we can rename the themes something like "Theme.OurAppNameWhichIDontKnow.EventDetails"?

File: app/src/main/java/com/connfa/schedule/ScheduleActivity.java [hunk 3]
```diff
@@ -59,8 +65,11 @@ protected void onStop() {
...
     protected Navigator navigate() {
-        return () -> {
-            // TODO
-        };
+        return navigator;
+    }
+
+    @Override
+    public void onEventClicked() {
+        navigate().toEventDetails();
```
Comment (tiwiz) [inline app/src/main/java/com/connfa/schedule/ScheduleActivity.java:73]: why not calling `navigator` itself in this case? (just curiosity)

File: app/build.gradle (hunk 1 above)
Comment (rock3r) [reply app/build.gradle:74]: That's a default value, branding is going to be on the droidconit17 branch

File: app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (rock3r) [reply app/src/main/AndroidManifest.xml:86]: We'll figure out naming later on

File: app/src/main/java/com/connfa/schedule/ScheduleActivity.java (hunk 3 above)
Comment (rock3r) [reply app/src/main/java/com/connfa/schedule/ScheduleActivity.java:73]: reads better, and keeps encapsulation, but the latter is a very weak reason :D

**Pull request merged**